packages/manager/modules/account-sidebar/src/user-infos/controller.js

```diff
@@ -45,7 +45,7 @@ export default class ManagerHubUserInfosCtrl {
   }
 
   getDisplayNicHandle() {
-    return toUpper(this.me.nichandle.replace(/-ovh$/, ''));
+    return this.me.nichandle;
```

Comment:
Is a function needed ? I agree it provides abstraction but in that case I think it would be better in a `User` model or somehting like that